integration-tests/_tests/04-multi-endpoint-test.test.ftd

```diff
@@ -13,7 +13,7 @@ fastn.assert.eq(fastn.http_response["data"], "Hello, World!");
 ;; Mountpoint: /ftd/* -> Endpoint: http://fastn.com/ftd/*
 -- fastn.get: Fetching content from fastn.com
 url: /ftd/column/
-http-status: 308
+http-status: 301
```

Comment:
Why this change? 307/308 are better replacements of 301/302.

Reply:
I did this because the CI was failing but not because of the changes introduced in this PR. I couldn't track down the change that caused it

Reply:
Got it.